### PR TITLE
dbvisualizer: 9.5.7 -> 10.0.25

### PR DESCRIPTION
**Pull request by piq9117**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [X] other Linux distributions (Ubuntu 18.04)
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/applications/misc/dbvisualizer/default.nix [hunk 1]
```diff
@@ -1,11 +1,11 @@
 { stdenv, fetchurl, jre, makeWrapper }:
 
 stdenv.mkDerivation {
-  name = "dbvisualizer-9.5.7";
+  name = "dbvisualizer-10.0.25";
```
Comment (jonringer) [inline pkgs/applications/misc/dbvisualizer/default.nix]:
```suggestion
  pname = "dbvisualizer";
  version = "10.0.25";
```

File: pkgs/applications/misc/dbvisualizer/default.nix [hunk 2]
```diff
@@ -1,11 +1,11 @@
 { stdenv, fetchurl, jre, makeWrapper }:
 
 stdenv.mkDerivation {
```
Comment (jonringer) [inline pkgs/applications/misc/dbvisualizer/default.nix]:
```suggestion
stdenv.mkDerivation rec {
```

File: pkgs/applications/misc/dbvisualizer/default.nix [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
-  name = "dbvisualizer-9.5.7";
+  name = "dbvisualizer-10.0.25";
 
   src = fetchurl {
-    url = https://www.dbvis.com/product_download/dbvis-9.5.7/media/dbvis_unix_9_5_7.tar.gz;
-    sha256 = "1xv4fw7cji2ffvv7z8vjl5lap512pj60s2ynihirrqld7pmklnyr";
+    url = https://www.dbvis.com/product_download/dbvis-10.0.25/media/dbvis_unix_10_0_25.tar.gz;
```
Comment (jonringer) [inline pkgs/applications/misc/dbvisualizer/default.nix]:
```suggestion
    url = https://www.dbvis.com/product_download/dbvis-${version}/media/dbvis_unix_${stdenv.lib.strings.replaceChars ["."] ["_"] version}.tar.gz;
```

**Review by jonringer — COMMENTED** [review]
allow for easy updates

**Comment by piq9117** [comment]
thanks @jonringer i'll be making those changes :smiley:

**Comment by jonringer** [comment]
mostly LGTM, remember to adhere to [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md) by squashing the small change commits

```
git reset HEAD^ # put last commit into unstaged
git add pkgs/
git commit --amend --no-edit
git push <fork> <branch> --force
```

**Comment by jonringer** [comment]
looks like i didn't get back around to this

do you mind rebasing on top of latest master?

**Pull request closed without merging**